solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -805,6 +806,12 @@ public void load() {
     containerHandlers.getApiBag().registerObject(clusterAPI);
     containerHandlers.getApiBag().registerObject(clusterAPI.commands);
     containerHandlers.getApiBag().registerObject(clusterAPI.configSetCommands);
+
+    if (this.isZooKeeperAware()) {
+      SchemaDesignerAPI schemaDesignerAPI = new SchemaDesignerAPI(this);
+      containerHandlers.getApiBag().registerObject(schemaDesignerAPI);
+    } // Schema Designer not available in standalone (non-cloud) mode
+
```

Comment:
Do you think the ideal programmatic experience would be for users in Standalone mode to receive this information (`not available in Standalone mode`) via API response?

Reply:
Not sure? The API is mostly intended to support the UI ... this feature needs UI and I don't foresee too many people wanting to interact with the Schema Designer API directly w/o using the Web Admin UI. The Schema Designer link in the Admin UI should not even appear in standalone mode.

Reply:
ahh. That makes sense. I wasn't sure if there would be two different personas interfacing, e.g, someone posting to a shared instance a schema they have ben toying with locally and another person reviewing the same schema.